assets/app/scripts/services/logLinks.js

```diff
@@ -50,10 +50,28 @@ angular.module('openshiftConsole')
         newTab(params);
       };
 
-      // Not currently used.
-      // new tab: path/to/current?view=textonly
-      var textOnlyLink = function() {
-        newTab({view: 'textonly'});
+      // broken up for readability:
+      var template = new URITemplate([
+        "/#/discover?",
+        "_g=(",
+          "time:(from:now-7y%2Fy,mode:relative,to:now)",
+        ")",
+        "&_a=(",
+          "columns:!(_source),",
+          "index:'{namespace}.*',",
+          "query:(",
+            "query_string:(",
+              "analyze_wildcard:!t,",
+              "query:'kubernetes_pod_name: {podname} %26%26 kubernetes_namespace_name: {namespace}'",
```

Comment:
Can we verify what we expect to actually get encoded for the fragment in the form? I have memories of spaces and percent encoding in fragments being super wonky across browsers